lib/ansible/modules/cloud/cloudstack/cs_vlan_ip_range.py

```diff
@@ -22,7 +22,9 @@
   network:
     description:
       - The network name or id.
-    required: true
+  physical_network:
+    description:
+      - The physical network name or id.
```

Comment:
~~~
    type: str
    version_added: '2.8'
~~~
I have a pending PR that fixes the other params.

Reply:
the whole module has `version_added: '2.8'`. Should it be explicitly stated that this param was added after the module was created?

Reply:
Hm, accodring to ansible docs it should be added if different than the overall `version_added`. I've added type specification.

Reply:
I thought the module was older :)